scout/server/blueprints/alignviewers/controllers.py

```diff
@@ -137,10 +143,7 @@ def make_sashimi_tracks(
     """
 
     locus = "All"
-
-    build = "38"
-    if "37" in str(case_obj.get("rna_genome_build", "38")):
-        build = "37"
+    build = get_case_genome_build(case_obj)
```

Comment:
Not really for the RNA genome build though!

Reply:
That'e exactly what reviews are for! 🤓